src/System.Device.Gpio/Interop/Unix/Libc/Interop.threads.cs

```diff
@@ -0,0 +1,63 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// Disable these StyleCop rules for this file, as we are using native names here.
+#pragma warning disable SA1300 // Element should begin with upper-case letter
+#pragma warning disable SA1307 // Field should begin with upper-case letter
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal static bool SetCurrentThreadHighPriority()
```

Comment:
I don't think our library should be changing thread priorities, that should be on user or at least be opt-in

Reply:
If possible, yes, this should be avoided.